api/v1alpha1/etcd_types.go

```diff
@@ -30,6 +30,11 @@ const (
 	Basic MetricsLevel = "basic"
 	// Extensive is a constant for metrics level extensive.
 	Extensive MetricsLevel = "extensive"
+
+	// VolumeRetentionPolicyDelete defines the delete policy for retaining old volumes
+	VolumeRetentionPolicyDelete VolumeRetentionPolicy = "Delete"
+	// VolumeRetentionPolicyRetain defines the retain policy for retaining old volumes
+	VolumeRetentionPolicyRetain VolumeRetentionPolicy = "Retain"
```

Comment:
Maybe this is something we should discuss. With `RetainMaster`, what would scale up look like for, say, a 3-node etcd cluster? Would we allow the 2 other etcd's to come up empty and then sync the data? Or would we follow our current approach of letting backup sidecar restore data on the other two nodes? If it's case 2, then I'm not sure we have much use for `RetainMaster`, since restoration is happening anyway on the other two etcd nodes. WDYT?

Reply:
I was thinking of case 1 (letting others nodes come up empty) but not a hard assumption. The main point was to rename `Retain` as `RetainAll` to make the behaviour clear. We can introduce `RetainMaster` (or better `RetainLeader`) later if it makes sense.